### PR TITLE
Show section parameter default value as inlay hint settings

### PR DESCRIPTION
**Pull request by angelozerr**
Show section parameter default value as inlay hint settings

See https://github.com/redhat-developer/quarkus-ls/issues/906

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -277,6 +277,11 @@
           "default": true,
           "markdownDescription": "Show section parameter type. Default is `true`."
         },
+        "qute.inlayHint.showSectionParameterType": {
```
Comment (datho7561) [inline package.json]: This key should be `qute.inlayHints.showSectionParameterDefaultValue` (currently there are two entries for `qute.inlayHint.showSectionParameterType`)

File: package.json [hunk 2]
```diff
@@ -277,6 +277,11 @@
...
+          "type": "boolean",
+          "default": true,
+          "markdownDescription": "Show section parameter type. Default is `true`."
```
Comment (datho7561) [inline package.json]: Update the description with what you wrote in the README

**Review by datho7561 — CHANGES_REQUESTED** [review]
Looks like the setting in the `package.json` doesn't line up with what you wrote in the `README`.

File: package.json (hunk 1 above)
Comment (angelozerr) [reply package.json]: Ooops sory, I forget to push my changed, it should be better

File: docs/qute/TemplateSupport.md [hunk 3]
```diff
@@ -112,12 +112,24 @@ CodeLens are clickable to jump to the proper Java type, method or field.
...
 Qute support displays the Java type of the parameter names as an inlay hint.
 
-![Inlay Hints for Qute](images/QuteTemplateInlayHint.gif)
+![Show Section parameter type](images/QuteTemplateInlayHint_ShowSectionParameterType.gif)
+
+To remove inlay hints on section parameter types only, disable `qute.inlayHints.showSectionParameterType` in settings.
+
+### Show Section parameter default value
+
+Qute support displays the default value of the parameter names as an inlay hint when user tag defines a default value (ex: `{#let name?="main.css"}`)
```
Comment (datho7561) [inline docs/qute/TemplateSupport.md:127]: Add a note that `qute.inlayHints.showSectionParameterType` also needs to be enabled for this to work

**Review by datho7561 — CHANGES_REQUESTED** [review]

File: docs/qute/TemplateSupport.md (hunk 3 above)
Comment (angelozerr) [reply docs/qute/TemplateSupport.md:127]: It is a bug!

Comment (angelozerr) [reply docs/qute/TemplateSupport.md:127]: Please retry it it should work now

Comment (angelozerr) [reply docs/qute/TemplateSupport.md:127]: Good catch!

Comment (datho7561) [reply docs/qute/TemplateSupport.md:127]: I still need `qute.inlayHints.showSectionParameterType` enabled for the default value inlay hint to appear. Is this expected?

Comment (angelozerr) [reply docs/qute/TemplateSupport.md:127]: You should have that :

QuteInlayHintSettings

Comment (datho7561) [reply docs/qute/TemplateSupport.md:127]: I'll try recompiling everything again, but it's not working for me.

Comment (datho7561) [reply docs/qute/TemplateSupport.md:127]: Okay, it works now!

**Review by datho7561 — APPROVED** [review]
Looks good. Thanks, Angelo!

**Pull request merged**